### DISCUSSION TITLE
Sorting documents Rest API

### DISCUSSION THREAD

**Discussion by magdalenapaluch** [outline/outline#2523] [Self Hosting]
Hi there,

It's great to have an option to sort the documents manually in the collection.

But I would like to fetch the data and be able to display it in the same order I sorted it manually.
Is there a way to do that?

**Comment by tommoor** [comment]
Hi, you can pass a `collectionId` and `sort="index"` to achieve this. It's only possible to sort this way inside the context of a collection

**Reply by magdalenapaluch** [reply]
I've already tried that, it's a bit strange, because when I pass "createdAt" or "updatedAt" to sort, it works fine, the items behave as I would expect. But when I do sort: "index", the page crashes and I get an "Error: internal_server_error". Do you have any idea what might be the issue here? Is that happening because I try to sort nested documents?

**Reply by tommoor** [reply]
Are you using `getoutline.com` or self hosted? If self hosted, which version? There should be error logs with more detail on the `500` error.
